x-pack/plugins/cases/public/components/case_action_bar/index.tsx

```diff
@@ -114,7 +114,7 @@ const CaseActionBarComponent: React.FC<CaseActionBarProps> = ({
             responsive={false}
             justifyContent="spaceBetween"
           >
-            {userCanCrud && !disableAlerting && (
+            {userCanCrud && syncAlertsFeatureOption && (
```

Comment:
Do we also need to check `ruleDetailsNavigation == null` here since that what the original logic was doing?

Reply:
I think that they are unrelated. The check here has to do with showing the sync alerts toggle in the single case view. If the `ruleDetailsNavigation` is null or not doesn't matter for this decision. You can still attach alerts to a case even with the `syncAlerts` set to false.